src/makeRobotAccountant.js

```diff
@@ -6,7 +6,19 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let count = 0;
+
+  const sum = (x) => {
+    count++;
+
+    if (count > 3 && count % 2 === 0) {
+      return () => 'Bzzz... Error!';
+    } else {
+      return (y) => x + y;
+    }
```

Comment:
Это условие можно записать без else
```suggestion
    }
    
    return (y) => x + y;
```

Reply:
так можна і без else